package.json

```diff
@@ -37,7 +37,8 @@
     "pino": "^8.17.2",
     "undici": "^6.3.0",
     "undici-retry": "^5.0.2",
-    "zod": "^3.22.4"
+    "zod": "^3.22.4",
+    "dot-prop": "6.0.1"
```

Comment:
can you add it to the dependabot ignore list?

Reply:
fixed